airbyte-config/config-models/src/main/resources/types/NormalizationDestinationDefinitionConfig.yaml

```diff
@@ -0,0 +1,21 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/NormalizationDestinationDefinitionConfig.yaml
+title: NormalizationDestinationDefinitionConfig
+description: describes a normalization config for destination definition
+type: object
+required:
+  - normalizationRepository
+  - normalizationTag
+  - normalizationIntegrationType
+additionalProperties: true
+properties:
+  normalizationRepository:
+    type: string
+    description: an optional field indicating the name of the repository to be used for normalization. If the value of the flag is NULL - normalization is not used.
```

Comment:
These fields are not optional here

Reply:
Yes you're right. For this object, these are mandatory fields. Thanks for noticing it.